content/en/blog/general/introducing-reports-server/index.md

```diff
@@ -67,7 +68,8 @@ Total size of etcd, including all resources in the cluster with respect to amoun
 
 ### With Reports Server
```

Comment:
Here, should we show the Postgres DB size as a separate column for completeness?